### ISSUE TITLE
Guru Meditation Error...

### ISSUE THREAD

**Issue by PaladinEng** [lucadentella/esp32-tutorial#1]
I've run this example, and while it does connect to my wifi, It also keeps resetting. Here's the log, backtrace is at the bottom. 

^[[0;32mI (29) boot: ESP-IDF v2.0-rc1-364-gcecdfdb 2nd stage bootloader^[[0m
^[[0;32mI (29) boot: compile time 10:03:11^[[0m
^[[0;32mI (63) boot: Enabling RNG early entropy source...^[[0m
^[[0;32mI (63) boot: SPI Speed      : 40MHz^[[0m
^[[0;32mI (64) boot: SPI Mode       : DIO^[[0m
^[[0;32mI (74) boot: SPI Flash Size : 4MB^[[0m
^[[0;32mI (86) boot: Partition Table:^[[0m
^[[0;32mI (98) boot: ## Label            Usage          Type ST Offset   Length^[[0m
^[[0;32mI (120) boot:  0 nvs              WiFi data        01 02 00009000 00006000^[[0m
^[[0;32mI (143) boot:  1 phy_init         RF data          01 01 0000f000 00001000^[[0m
^[[0;32mI (166) boot:  2 factory          factory app      00 00 00010000 00100000^[[0m
^[[0;32mI (190) boot: End of partition table^[[0m
^[[0;32mI (203) boot: Disabling RNG early entropy source...^[[0m
^[[0;32mI (220) boot: Loading app partition at offset 00010000^[[0m
^[[0;32mI (940) boot: segment 0: paddr=0x00010018 vaddr=0x00000000 size=0x0ffe8 ( 65512) ^[[0m
^[[0;32mI (941) boot: segment 1: paddr=0x00020008 vaddr=0x3f400010 size=0x08424 ( 33828) map^[[0m
^[[0;32mI (957) boot: segment 2: paddr=0x00028434 vaddr=0x3ffb0000 size=0x02494 (  9364) load^[[0m
^[[0;32mI (987) boot: segment 3: paddr=0x0002a8d0 vaddr=0x40080000 size=0x00400 (  1024) load^[[0m
^[[0;32mI (1010) boot: segment 4: paddr=0x0002acd8 vaddr=0x40080400 size=0x1a7f8 (108536) load^[[0m
^[[0;32mI (1088) boot: segment 5: paddr=0x000454d8 vaddr=0x400c0000 size=0x00000 (     0) load^[[0m
^[[0;32mI (1089) boot: segment 6: paddr=0x000454e0 vaddr=0x00000000 size=0x0ab28 ( 43816) ^[[0m
^[[0;32mI (1105) boot: segment 7: paddr=0x00050010 vaddr=0x400d0018 size=0x3abf4 (240628) map^[[0m
^[[0;32mI (1131) cpu_start: Pro cpu up.^[[0m
^[[0;32mI (1143) cpu_start: Starting app cpu, entry point is 0x40080bb4^[[0m
^[[0;32mI (0) cpu_start: App cpu up.^[[0m
^[[0;32mI (1176) heap_alloc_caps: Initializing. RAM available for dynamic allocation:^[[0m
^[[0;32mI (1198) heap_alloc_caps: At 3FFAE2A0 len 00001D60 (7 KiB): DRAM^[[0m
^[[0;32mI (1219) heap_alloc_caps: At 3FFB7528 len 00028AD8 (162 KiB): DRAM^[[0m
^[[0;32mI (1240) heap_alloc_caps: At 3FFE0440 len 00003BC0 (14 KiB): D/IRAM^[[0m
^[[0;32mI (1261) heap_alloc_caps: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM^[[0m
^[[0;32mI (1283) heap_alloc_caps: At 4009ABF8 len 00005408 (21 KiB): IRAM^[[0m
^[[0;32mI (1304) cpu_start: Pro cpu start user code^[[0m
^[[0;32mI (1365) cpu_start: Starting scheduler on PRO CPU.^[[0m
^[[0;32mI (212) cpu_start: Starting scheduler on APP CPU.^[[0m
^[[0;32mI (252) phy: phy_version: 350, Mar 22 2017, 15:02:06, 1, 0^[[0m
Connecting to MYSSID
Guru Meditation Error of type LoadProhibited occurred on core  1. Exception was unhandled.
Register dump:
PC      : 0x40083872  PS      : 0x00060930  A0      : 0x80081a10  A1      : 0x3ffc3f00
A2      : 0xa5a5a5a5  A3      : 0xffffffff  A4      : 0x00060920  A5      : 0x3ffc4730
A6      : 0x00000000  A7      : 0x00000000  A8      : 0x80083048  A9      : 0x3ffc3ef0
A10     : 0x00000003  A11     : 0x00060923  A12     : 0x00060920  A13     : 0x00000000
A14     : 0x00000000  A15     : 0x00000000  SAR     : 0x00000000  EXCCAUSE: 0x0000001c
EXCVADDR: 0xa5a5a5a9  LBEG    : 0x400014fd  LEND    : 0x4000150d  LCOUNT  : 0xffffffff

Backtrace: 0x40083872:0x3ffc3f00 0x40081a10:0x3ffc3f20 0x40081af0:0x3ffc3f50 0x400fb281:0x3ffc3f70 0x400fe466:0x3ffc4280 0x400fb324:0x3ffc43a0 0x400f4f20:0x3ffc46b0 0x40107d8d:0x3ffc4700

**Comment by tgarc** [comment]
I'm assuming this is for the 02_wifi_connection example? If so I'm encountering the same issue...

**Comment by tgarc** [comment]
Ah, okay so the solution for this is mentioned in the comments of one of the tutorials: http://www.lucadentella.it/en/2017/01/16/esp32-6-collegamento-ad-una-rete-wifi/

Simply comment out the line
```
setvbuf(stdout, NULL, _IONBF, 0);
```

and it should work. I also had to add a line to flush the output. Right before the infinite loop I added

```
fflush(stdout);
```

And I had to reset the esp chip also. But then it worked like a charm!
